### PR TITLE
make awxkit have the same version as the AWX package

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awxkit/setup.py [hunk 1]
```diff
@@ -53,7 +60,7 @@ def run(self):
 version = '0.1.0'
```
Comment (shanemcd) [inline awxkit/setup.py]: Does this need to 🚫 ?

Comment (ryanpetrello) [reply awxkit/setup.py]: Yup, good eye.

**Review by shanemcd — APPROVED** [review]

**Comment by chrismeyersfsu** [comment]
Does a pip install of the devel branch w/ this subdir work in this case?

**Comment by ryanpetrello** [comment]
@chrismeyersfsu should work just fine?  They're two distinct packages:

image

The main awx package:

image

**Comment by chrismeyersfsu** [comment]
K. I was just wondering if pip only somehow checked out the subdir that the awx-cli package is in. In which case the symbol link to the VERSION could be problematic. But it sounds like it clones the entirety :+1:

**Review by chrismeyersfsu — APPROVED** [review]

**Pull request merged**